server/registry/internal/storage/list.go

```diff
@@ -109,6 +110,51 @@ var artifactFields = map[string]filtering.FieldType{
 	"size_bytes":  filtering.Int,
 }
 
+func gormOrdering(ordering string, fields map[string]filtering.FieldType) (string, error) {
```

Comment:
Could we add  a brief comment here explaining what this is doing?

something like: "parses specified order_by values, generating a string that gets added to the gorm queries that implements the specified ordering"

Reply:
Done.